configuration/pih/drugs/drugs.csv

```diff
@@ -422,77 +477,80 @@ c6a94776-fce4-11e9-8f0b-362b9e155667,18228,"Rifampicin (R) 75mg + Isoniazid (H)
 f55edf29-b1ac-4c43-8fb4-57a9c4415c0c,11806,"Streptomycin sulfate, Powder for solution for injection, 1g vial",CIEL:84361,
-fb8c642a-8c4d-4442-bfbc-d41308c07c41,11031,"Suxamethonium chloride, Solution for injection, 50mg/mL, 2mL ampoule",CIEL:84583,
-8112cd32-ae2a-11e4-ab27-0800200c9a66,11838,"Tamoxifen, 20mg tablet",PIH:7664,
+bf59d3b9-d744-445b-8e3b-f0e7e9f0767b,10352,"Sulfadoxine (S) 500mg + Pyrimethamine (P) 25mg tablet",PIH:12344,
+fb8c642a-8c4d-4442-bfbc-d41308c07c41,12591,"Suxamethonium chloride, Solution for injection, 50mg/mL, 2mL ampoule",CIEL:84583,
+07e34767-497b-4dbe-883a-1520d90364f8,12268,"Tamsulosin, 400 microgram tablet",PIH:12170,
+8112cd32-ae2a-11e4-ab27-0800200c9a66,12605,"Tamoxifen, 20mg tablet",PIH:7664,
 78f96fda-dfbe-11e9-8a34-2a2ae2dbcce4,10856,"Tenofovir disoproxil fumarate (TDF) 300mg + Lamivudine (3TC) 300mg + Efavirenz (EFV) 600mg tablet",CIEL:164505,
 78f96a76-dfbe-11e9-8a34-2a2ae2dbcce4,10706,"Tenofovir disoproxil fumarate (TDF) 300mg + Lamivudine (3TC) 300mg tablet",CIEL:161364,
 78faa576-dfbe-11e9-8a34-2a2ae2dbcce4,11870,"Tenofovir disoproxil fumarate (TDF), 300mg tablet",CIEL:84795,
 a40d9ebc-c981-11e7-abc4-cec278b6b50a,10556,"Terbinafine hydrochloride, 250mg tablet",CIEL:84825,
 737a2d0f-82f9-4e1d-887d-c28ab3728502,11515,"Tetanus antitoxin (Equine), 1,500 IU/mL, 1mL ampoule",CIEL:84877,
-e66b74f7-854e-4392-852a-886b1d31d9fc,11259,"Tetracaine hydrochloride, Eye drop, 0.5%, 10mL bottle",CIEL:84889,
-d9333e11-4b28-41fe-93c5-44e624ef1b2c,10352,"Tetracycline hydrochloride, Eye ointment, 1%, 5 gram tube",CIEL:84894,
+e66b74f7-854e-4392-852a-886b1d31d9fc,11029,"Tetracaine hydrochloride, Eye drop, 0.5%, 10mL bottle",CIEL:84889,
+d9333e11-4b28-41fe-93c5-44e624ef1b2c,11121,"Tetracycline hydrochloride, Eye ointment, 1%, 5 gram tube",CIEL:84894,
 44ad6301-71e6-453d-80c1-fe0d409dc63c,10272,"Thiamine (Vitamin B1), 100mg tablet",CIEL:84995,
+30016839-8b3d-4eea-a647-0d8342e44ddb,12129,"Thiamine (Vitamin B1), 50mg tablet",CIEL:84995,
 35816ccb-904b-4ca0-8160-d7de356d5f1e,11801,"Thiamine (Vitamin B1), Solution for injection, 50mg/mL, 2mL ampoule",CIEL:84995,
-88c829e2-081b-4082-9185-79928a39f305,10219,"Thiopental, Powder for solution for injection, 500mg vial",CIEL:85015,
-25da272a-5892-4c53-87bd-96c6a2d8689f,11029,"Timolol maleate, Eye drop, 0.25%, 5mL bottle",CIEL:85134,
-48bad836-e456-40db-be79-769e2798b23e,11121,"Tramadol hydrochloride, 50mg capsule",CIEL:85276,
-43271fad-6f01-4a94-9e4f-1746642a3af6,10501,"Tramadol hydrochloride, Solution for injection, 50mg/mL, 2mL ampoule",CIEL:85276,
-8319e5d6-777e-4d0e-9aad-099e684dcc8e,11555,"Tranexamic acid, Solution for injection, 100mg/mL, 5mL ampoule",CIEL:85286,
+88c829e2-081b-4082-9185-79928a39f305,11069,"Thiopental, Powder for solution for injection, 500mg vial",CIEL:85015,
+25da272a-5892-4c53-87bd-96c6a2d8689f,10766,"Timolol maleate, Eye drop, 0.25%, 5mL bottle",CIEL:85134,
+48bad836-e456-40db-be79-769e2798b23e,10165,"Tramadol hydrochloride, 50mg capsule",CIEL:85276,
```

Comment:
This seems wrong.  https://obnav.pih-emr.org/openboxes/inventoryItem/showStockCard/11121